data/Akhil2707.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "Akhil Rathipelly",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Iam a full stack developer",
+  "avatar": "https://github.com/Akhil2707.png",
```

Comment:
In our recent update, we no longer need `avatar` key, it will auto-pick your GitHub Profile. So Please remove avatar field from your file. 

Updated docs: https://linkfree.eddiehub.io/docs/quickstart

```suggestion
```

Reply:
removed avatar ,please look into this merge the pr if changes are needed let me know ,thank you